### PR TITLE
1. Shell and python file to only download image chat data. 2. Updated…

### PR DESCRIPTION
**Pull request by shubhamagarwal92**
**Patch description**
1. Shell and python file to only download `image_chat` data. 
2. Updated Readme for the same.

The current setup to download just the data is cumbersome requiring full knowledge of ParlAI platform. Making it easy for researchers who just want to use the data and build their code on top of different repos. Basically a direct api for the `build` in `parlai.tasks.image_chat.build`

**Testing steps**
Enter steps to test your pull request. Give a clear and concise description of
what you expected to happen during testing.

Run as `./parlai/tasks/image_chat/download_data.sh` to download only data. (Updated the README.) 

**Logs**
```
Running code from: 
Downloading in data root:
[ downloading: http://parl.ai/downloads/image_chat/image_chat.tgz to .....]
```

**Other information**
@klshuster please verify if this is the correct data.

### CODE REVIEW

**Comment by stephenroller** [comment]
Thanks for your contribution. The unit tests failures look transient, so I've hit rerun. Definitely please fix the lint error. I'll leave it to @klshuster to evaluate the contents of the PR.

File: parlai/tasks/image_chat/download_data.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+export CURRENT_DIR=${PWD}
+# Should ideally give the parlai clone path
+echo "Running code from: " $CURRENT_DIR
+
+#export TASK_DIR="$(dirname "$CURRENT_DIR")"
```
Comment (klshuster) [inline parlai/tasks/image_chat/download_data.sh]: nit: could you perhaps remove these options if they are not being used?

**Comment by shubhamagarwal92** [comment]
Hi @klshuster 

I have added the copyright and run pylint. Please find the output now

```
pylint download_data.py

************* Module parlai.tasks.image_chat.download_data
download_data.py:1:0: C0114: Missing module docstring (missing-module-docstring)
download_data.py:29:4: C0103: Constant name "opt" doesn't conform to UPPER_CASE naming style (invalid-name)

------------------------------------------------------------------
Your code has been rated at 8.18/10 (previous run: 6.36/10, +1.82)
```

**Comment by klshuster** [comment]
could you please run `bash autoformat.sh` in the ParlAI repo and then push those changes?

**Comment by shubhamagarwal92** [comment]
@klshuster Seems like autoformat doesn't like the location of the files. I don't know what is happening under the hood but this commit https://github.com/facebookresearch/ParlAI/pull/2381/commits/cd8a0e41c2ece71968c4cb8ea52bc95aee96f741 shows that yml files have somehow been changed, due to which the tests are failing. 

Do you want me to manually correct the yml and md files?

**Comment by klshuster** [comment]
yes, could you please revert anywhere where it changed "setup.py" to "download_data.py"

**Comment by klshuster** [comment]
Not sure why `autoformat.sh` won't work for you, however here are the lint changes to make, then we should be good to merge:

```
 def parse_args():
     """
-    Wrapper to parse CLI arguments
+    Wrapper to parse CLI arguments.
+
     :return: args
     """
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "-dp", "--datapath", default="/tmp",
-        help="Path where to save data."
+        "-dp", "--datapath", default="/tmp", help="Path where to save data."
     )
```

**Comment by shubhamagarwal92** [comment]
Thanks for your suggestion. :)

**Comment by klshuster** [comment]
I'll go ahead and merge this

**Pull request merged**